Composer/packages/extensions/obiformeditor/src/Form/fields/PromptField/PromptSettings.tsx

```diff
@@ -19,52 +28,77 @@ interface PromptSettingsrops extends FieldProps<MicrosoftInputDialog> {
 
 export const PromptSettings: React.FC<PromptSettingsrops> = props => {
   const { formData, idSchema, getSchema, onChange, errorSchema } = props;
+  const [errorMessage, setErrorMessage] = useState(false);
 
+  const onValidate = errMessage => {
+    setErrorMessage(errMessage);
+  };
   return (
-    <div css={settingsFields}>
-      <div css={[field, settingsFieldHalf]}>
-        <TextWidget
-          onChange={onChange('maxTurnCount')}
-          schema={getSchema('maxTurnCount')}
-          id={idSchema.maxTurnCount.__id}
-          value={formData.maxTurnCount}
-          label={formatMessage('Max turn count')}
-          formContext={props.formContext}
-          rawErrors={errorSchema.maxTurnCount && errorSchema.maxTurnCount.__errors}
-        />
+    <div>
+      <div css={settingsFields}>
+        <div css={[field, settingsFieldHalf]}>
+          <TextWidget
+            onChange={onChange('maxTurnCount')}
+            schema={getSchema('maxTurnCount')}
+            id={idSchema.maxTurnCount.__id}
+            value={formData.maxTurnCount}
+            label={formatMessage('Max turn count')}
+            formContext={props.formContext}
+            rawErrors={errorSchema.maxTurnCount && errorSchema.maxTurnCount.__errors}
+            hiddenErrMessage={true}
+            onValidate={onValidate}
+          />
+        </div>
+        <div css={[field, settingsFieldHalf]}>
+          <TextWidget
+            onChange={onChange('defaultValue')}
+            schema={getSchema('defaultValue')}
+            id={idSchema.defaultValue.__id}
+            value={formData.defaultValue}
+            label={formatMessage('Default value')}
+            formContext={props.formContext}
+            rawErrors={errorSchema.defaultValue && errorSchema.defaultValue.__errors}
+            hiddenErrMessage={true}
+            onValidate={onValidate}
+          />
+        </div>
```

Comment:
Let's create a new form field that renders two schema fields and handles the error presentation.